sdk/devcenter/Azure.Developer.DevCenter/README.md

```diff
@@ -36,16 +36,11 @@ To use Azure Active Directory authentication, add the Azure Identity package:
 You will also need to register a new AAD application, or run locally or in an environment with a managed identity.
 If using an application, set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
 
-```
-Uri endpoint = new Uri("<dev-center-uri>");
-var client = new DevCenterClient(endpoint, new DefaultAzureCredential());
-```
-
```

Comment:
What was the reason for removing this snippet? We need to keep this snippet in this section to demonstrate how the client is created.

Please see an example here: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/eventgrid/Azure.Messaging.EventGrid#authenticate-using-azure-active-directory

Reply:
Also, can we create a snippet for it? something like `Snippet:Azure_DevCenter_CreateClient`

Reply:
Added the snippet.